File: test_repro.test.js
// Function definition
function calculate() {
  return 6 * 7;
}

// Jest tests
describe('Test repro', () => {
  test('calculate', () => {
    expect(calculate()).toBe(42);
  });
  
  test('other', () => {
    expect(calculate()).toBe(43);
  });
});

module.exports = { calculate };

File: test/test_repro_test.dart
import 'package:test/test.dart';

int calculate() {
  return 6 * 7;
}

void main() {
  group('Test repro', () {
    test('calculate', () {
      expect(calculate(), equals(42));
    });
    test('other', () {
      expect(calculate(), equals(43));
    });
  });
}
